windows/src/test/unit-tests/kmcomp/test_casedkeys_mnemonic_3.kmn

```diff
@@ -0,0 +1,9 @@
+c Tests mnemonic layout without any casedkeys
+store(&NAME) 'Test mnemonic layout but no cased keys'
+store(&mnemoniclayout) '1'
```

Comment:
Should there be a test case for mnemonic and cased keys?

Reply:
See test_casedkeys_mnemonic_2.kmn and test_casedkeys_mnemonic_1.kmn